src/calliope_pathways/model_configs/italy_stationary/model.yaml

```diff
@@ -12,7 +10,7 @@ config:
     # What version of Calliope this model is intended for
     calliope_version: 0.7.0
     # Activate pathways configuration
-    add_math: [../../math/pathways.yaml]
+    add_math: [../../math/pathways.yaml, math.yaml]
```

Comment:
I suspect we'll add more math, so let's move this into pathways/math/import_export.yaml ?

Reply:
Since it is specific to the `italy` model, I wanted to stay with it.

Reply:
Ahhh, I get it. It's fine, then. We can rename this file if we introduce more math options.